modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/DotExpanderProcessorTests.java

```diff
@@ -46,7 +46,7 @@ public void testEscapeFields() throws Exception {
         document = new IngestDocument(source, Collections.emptyMap());
         processor = new DotExpanderProcessor("_tag", null, null, "foo.bar");
         processor.execute(document);
-        assertThat(document.getSourceAndMetadata().size(), equalTo(1));
+        assertThat(document.getSourceAndMetadata().size(), equalTo(2));
```

Comment:
Why did the map size change?

Reply:
_version is added in the test constructor so it doesn't have to be added in every test that uses IngestDocument.

Reply:
This seems trappy. Just above here the IngestDocument is created. If we need helpers to not have to specify version or other metadata fields, let's add helper methods, but not make it magical. If I were debugging this code, I would be very confused why I see 2 here.